UPGRADE-4.4.md

```diff
@@ -106,7 +106,7 @@ Lock
 
  * Deprecated `Symfony\Component\Lock\StoreInterface` in favor of `Symfony\Component\Lock\BlockingStoreInterface` and
    `Symfony\Component\Lock\PersistingStoreInterface`.
- * `Factory` is deprecated, use `LockFactory` instead
+ * `Symfony\Component\Lock\Factory` is deprecated, use `Symfony\Component\Lock\LockFactory` instead
```

Comment:
why FQCN here and not elsewhere? should we revert here? if not, don't miss updating also UPGRADE-5.0.md

Reply:
It seems that we use FQCN only for interfaces, will revert.